src/views/Simulation/utils/graphUtils.js

```diff
@@ -61,7 +61,35 @@ const getGraphLinks = (instance, nodes) => {
   return [...getTransportLinks(instance, nodes), ...getInputLinks(instance, nodes), ...getOutputLinks(instance, nodes)];
 };
 
-export const getGraphFromInstance = (instance, settings) => {
+const setResourceBottlenecks = (instance, productionResources, operations, bottlenecks) => {
+  let operationsNotFound = 0;
+  let parentResourcesNotFound = 0;
+  for (const [operation, operationBottlenecks] of Object.entries(bottlenecks)) {
```

Comment:
Commonly, I prefer use `map` or `forEach` functions. They offer more security in execution.
To make `continue` inside a `forEach` function, you can use `return`.